libr/core/cmd_print.c

```diff
@@ -625,7 +625,7 @@ static const ut32 colormap[256] = {
 
 static void __cmd_pad(RCore *core, const char *arg) {
 	if (*arg == '?') {
-		eprintf ("Usage: pad [hexpairs] # disassembly given bytes\n");
+		R_LOG_ERROR ("Usage: pad [hexpairs] # disassembly given bytes");
```

Comment:
this is a help message not an error. you should replace this code with the RCoreHelp api instead

Reply:
will come back to that case later. Just saw a lot of eprintf/r_log_error usages over the ("Usage: ...). Will do that in another PR